front-end/src/Header.js

```diff
@@ -0,0 +1,25 @@
+import 'react-sticky-header/styles.css';
+import StickyHeader from 'react-sticky-header';
+import './Header.css' 
+import Hamburger from './Hamburger'
+
+const Header = () => (
+  [
```

Comment:
I think this definition for Header is a bit wrong--it should be of the format `const Header = () => { 
return (<Hamburger> and other components) }`. Right now as-is, React thinks it's a list because of the `[]` surrounding the components.

Reply:
I'm not entirely sure how to do this and I keep messing it up every time I try. Would you be able to help me do this on Sunday?